apaq-v1.0.yaml

```diff
@@ -0,0 +1,57 @@
+openapi: 3.0.0
+
+info:
+  title: Airport Air Quality Prediction API
+  version: 1.0.0
+  description: API for predicting air quality based on airport information
+
+basePath: /v1
+
+paths:
+  /predict-air-quality:
+    post:
+      summary: Predict Air Quality from Airport Information
+      description: |
+        Uploads airport information and predicts air quality based on the provided data.
+        Initiates background processing for prediction.
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                altitude:
+                  type: number
+                  description: Altitude of the airport
+                runways:
+                  type: array
+                  description: Information about runways
```

Comment:
The description should be self explanatory and tell me what this field actually is. Something `A list of available runways at the specified airport` would be much better